src/utils/github.ts

```diff
@@ -0,0 +1,127 @@
+import { Octokit } from 'octokit';
+import axios from 'axios';
+import pLimit from 'p-limit';
+
+import { log } from '../log.js';
+
+import path from 'path';
+import fs from 'fs';
+
+const MULTITHREADING_LIMIT = 10;
+
+export async function getContent(owner: string, repository: string) {
+  const octokit = new Octokit({});
+  return (
+    await octokit.rest.repos.getContent({
+      owner: owner,
+      repo: repository,
+    })
+  ).data;
+}
+
+export async function getFileContent(
+  owner: string,
+  repository: string,
+  path: string,
+) {
+  const url =
+    'https://raw.githubusercontent.com/' +
+    owner +
+    '/' +
+    repository +
+    '/main/' +
+    path;
```

Comment:
Can we extract this into a util function that builds the URL? Since it's used in multiple place

Reply:
Extracted into own util function 👍